### PR TITLE
DocBlock: `tribe_create_venue()`

### PR DESCRIPTION
**Pull request by sdokus**
Minor tweak to add a legacy note to docblock for `tribe_create_venue()`

### CODE REVIEW

**Comment by sdokus** [comment]
I need to add the readme.txt changes

File: readme.txt [hunk 1]
```diff
@@ -234,6 +234,7 @@ Remember to always make a backup of your database and files before updating!
 
 = [6.5.1] TBD =
 
+* Tweak - Add note to `tribe_create_venue()` docblock to indicate future deprecation.
```
Comment (stratease) [inline readme.txt:237]:
```suggestion
* Tweak - Add note to `tribe_create_venue()`, `tribe_create_event()`.... docblock to indicate future deprecation.
```
Maybe we consolidate these changelog entries into one when you're done.

**Review by stratease — APPROVED** [review]

File: readme.txt (hunk 1 above)
Comment (bordoni) [reply readme.txt:237]: I like that.

File: readme.txt [hunk 2]
```diff
@@ -234,6 +234,7 @@ Remember to always make a backup of your database and files before updating!
...
 * Tweak - Add note to `tribe_create_organizer()` docblock to indicate future deprecation.
 * Tweak - Add note to `tribe_create_event()` docblock to indicate future deprecation.
 * Tweak - Add note to `tribe_event_update()` docblock to indicate future deprecation.
```
Comment (Camwyn) [inline readme.txt:240]:
```suggestion
* Tweak - Add note to `tribe_create_venue()`, `tribe_create_organizer()`, `tribe_create_event()`, and `tribe_event_update()` docblocks to indicate future deprecation.
```

Comment (Camwyn) [reply readme.txt:240]: Consolidate them into one entry.

**Review by Camwyn — CHANGES_REQUESTED** [review]

**Review by codingmusician — DISMISSED** [review]

File: src/functions/advanced-functions/venue.php [hunk 3]
```diff
@@ -28,12 +30,14 @@
...
+	 *
 	 * @param array $args Elements that make up post to insert.
 	 *
-	 * @return int ID of the Venue that was created. False if insert failed.
-	 * @link     http://codex.wordpress.org/Function_Reference/wp_insert_post
 	 * @see      wp_insert_post()
-	 * @category Venues
+	 * @link     http://codex.wordpress.org/Function_Reference/wp_insert_post
+	 *
+	 * @return int ID of the Venue that was created. False if insert failed.
```
Comment (bordoni) [inline src/functions/advanced-functions/venue.php:40]:
```suggestion
	 * @return int|false ID of the Venue that was created. False if insert failed.
```

**Review by bordoni — CHANGES_REQUESTED** [review]

**Review by bordoni — APPROVED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**